easy/38-count-and-say.js

```diff
@@ -0,0 +1,82 @@
+/* Time/Space ? (would love some input on this) - I believe this is O(N^2) on time
```

Comment:
I'm not entirely sure what this code does but yeah I'm 90% sure this would be O(N^2)

Reply:
The countSimultaneousOccurances is O(N) inside the loop after countAndSay's recursive call, so yeah I believe so, unless the recursive call bumps it up